Check for internet connection added :boom:

tarungarg546 opened the pull request:
Hey @hemanth ,

This PR handles the case when `user` has no internet connection.

We can check its status simply by checking if we can get `DNS` of a url or not .

tarungarg546 committed:
Check for internet connection added :boom:

hemanth commented on cli.js:
```diff
@@ -15,4 +15,11 @@ if(cli.input.length === 0) {
 	process.exit(0);
 }
 
+dns.lookup('www.google.com', err => {
```
Nice, how about using `is-online` module?

tarungarg546 replied:
Yeah, that can be good option too.
Will look into that.

tarungarg546 replied:
Done :smile:

tarungarg546 committed:
Added is-online module

Is-online module will help us to detect internet connectivity issue in user land

tarungarg546 committed:
Arrow function
